query-service-impl/src/test/resources/prometheus_request_handler.conf

```diff
@@ -0,0 +1,36 @@
+{
+  name = raw-service-view-service-prometheus-handler
+  type = prometheus
+  clientConfig = ""
+  requestHandlerInfo {
+    tenantAttributeName = tenant_id
+    startTimeAttributeName = "SERVICE.startTime"
+    prometheusViewDefinition {
+      viewName = rawServiceView
+      metricScope = SERVICE
+      metricMap {
+        numCalls {
+          metricName: "num_calls",
+          metricType: "GAUGE"
+        },
+        errorCount {
```

Comment:
we only have num_calls emitter for now. So, can you remove this? We will add it later when we add error_count emitter.